mlflow/pyfunc/__init__.py

```diff
@@ -265,7 +268,58 @@ def _load_model_env(path):
     return _get_flavor_configuration(model_path=path, flavor_name=FLAVOR_NAME).get(ENV, None)
 
 
-def load_model(model_uri, suppress_warnings=True):
+PyFuncOutput = Union[pandas.DataFrame, pandas.Series, np.ndarray, list]
+
+
+class PyFuncModel(object):
+    """
+    MLflow 'python function' model.
+
+    Wrapper around model implementation and metadata. This class is not meant to be constructed
+    directly. Instead, instances of this class are constructed and returned from
+    py:func:`mlflow.pyfunc.load_model`.
+
+    ``model_impl`` can be any Python object that implements the `Pyfunc interface
+    <https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#pyfunc-inference-api>`_, and is
+    by invoking the model's ``loader_module``.
+    
+    ``model_meta`` is loaded by `:py:func:load_model` from the model metadata (MLmodel) file
```

Comment:
Somewhere can we link to what model_meta is? Is it just the JSON form of the MLmodel file or some other type?

Reply:
It's the Model object. I'll update the docstring.